v2/samples/documentdb/mongodb/v1beta/v1beta20210515_mongodbdatabase.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: documentdb.azure.com/v1beta20210515
 kind: MongodbDatabase
 metadata:
-  name: sample-db-mongodb
+  name: sample-db-mongodb-beta
```

Comment:
Why do we need this?

Reply:
We have name randomization turned off for documentdb (not entirely sure why). So the two samples alpha/beta having the same names means they conflict when you try to re-record them both at the same time.

Changing the name means that you can re-record both samples at the same time which makes things faster.